js/okex.js

```diff
@@ -694,6 +694,53 @@ module.exports = class okex extends Exchange {
         return this.safeInteger (first, 'ts');
     }
 
+    async fetchAccounts (params = {}) {
+        const response = await this.privateGetAccountConfig (params);
+        //
+        //     {
+        //         "code": "0",
+        //         "data": [
+        //             {
+        //                 "acctLv": "2",
+        //                 "autoLoan": false,
+        //                 "ctIsoMode": "automatic",
+        //                 "greeksType": "PA",
+        //                 "level": "Lv1",
+        //                 "levelTmp": "",
+        //                 "mgnIsoMode": "automatic",
+        //                 "posMode": "long_short_mode",
+        //                 "uid": "88018754289672195"
+        //             }
+        //         ],
+        //         "msg": ""
+        //     }
+        //
+        const data = this.safeValue (response, 'data', []);
+        const result = [];
+        for (let i = 0; i < data.length; i++) {
+            const account = data[i];
+            const accountId = this.safeString (account, 'uid');
+            let accountLevel = undefined;
+            const type = this.safeString (account, 'acctLv');
+            if (type === '1') {
+                accountLevel = 'Simple';
+            } else if (type === '2') {
+                accountLevel = 'Single-currency margin';
+            } else if (type === '3') {
+                accountLevel = 'Multi-currency margin';
+            } else {
+                accountLevel = 'Portfolio margin';
```

Comment:
Where do we get these accountLevel strings from? If they're made up by us, then I would suggest to use the exchange-specific value of type.

Reply:
These are from the description of account level from the api docs:

acctLv | String | Account level 1: Simple 2: Single-currency margin 3: Multi-currency margin 4：Portfolio margin
-- | -- | --

https://www.okx.com/docs-v5/en/#rest-api-account-get-account-configuration

Let me know if I should leave it as is, return the acctLv number, or set the type to undefined.

Reply:
I suggest to use acctLv number as a string-type.